simtools/io_operations/io_hdf5.py

```diff
@@ -0,0 +1,112 @@
+import logging
+from pathlib import PosixPath
+
+import astropy.units as u
+import tables
+from astropy.table import Table
+from ctapipe.io import read_table
+
+from simtools.utils.names import sanitize_name
+
+__all__ = [
+    "fill_hdf5_table",
+    "read_hdf5",
+]
+
+_logger = logging.getLogger(__name__)
+
+
+def fill_hdf5_table(hist, x_bin_edges, y_bin_edges, x_label, y_label, meta_data):
```

Comment:
It is good to have all the hdf5 functions together, but this function is not actually an io as it does not deal with input and output. Instead, if fills the table. Maybe we should call the module `hdf5_handler` instead?

Reply:
Thanks - this is a good suggestion. I've renamed `io_hdf5` to `hd5_handler`.

Waiting for the tests and will merge when successful.